blog/2015/12/three-flavors-of-instructor-training.html

```diff
@@ -0,0 +1,41 @@
+---
+layout: blog
+root: ../../..
+author: "Stephen Crouch, Christina Koch, Karin Lagesen, Aleksandra Pawlik, Fiona Tweedie, and Greg Wilson"
+title: "Three Flavors of Instructor Training"
+date: 2015-12-11
+time: "01:00:00"
+category: ["Instructor Training"]
+---
+* in-person in Manchester
+* multi-week fully-distributed
+  * meeting every week instead of every second week (like we've done in the past)
+  * 54 started, 48 stayed with us 'til the end (highest completion rate we've had in two years)
+  * used Piazza for managing the class - not noticeably better than a shared WordPress blog, but easier on many people than our experiment with GitHub earlier this year
+  * used BlueJeans for meetings (each meeting held twice to accommodate time zones - BlueJeans mostly worked, but sometimes had trouble with # of participants)
+  * started training other trainers (plus a guest speaker - hi, Sarah
+* two-day group-based
+  * 9 groups handled by 4 trainers  (1 / 3 / 2 / 3 groups per instructor based primarily on geography)
+  * team sizes were:
+    * Townsville: 4
+    * Potsdam + Paris + Thessaloniki: ??
+    * Toronto (14) + Curitiba (3) = 17
+    * Wisconsin + Arlington + Vancouver: ??
+  * used one Google Hangout for each of the four groups
+    * occasional glitches, but mostly reliable
+    * DDoS attack on UK academic networks didn't help - lots of things inaccessible, but we managed
+* overall
+  * mostly positive, and certainly less total effort for instructors than the multi-week version
+  * some participants may have been less enthusiastic than usual (roped in by friends to get numbers)
+  * not much inter-group discussion (sessions tended to be hub-and-spoke with the trainer as the hub)
+  * harder to give good feedback (esp. on practice teaching) over the web than in person
+  * The live coding was a really good exercise, I think particularly for the more experienced coders - made them slow down
+* individual
+  * Greg:
+    * mornings went better than afternoons
+      * that's always the case, but being 5 hours ahead of one of my groups meant I was teaching in my afternoon and evening
+  * Fiona:
+    * online live course made a lot more use of the Etherpad, which meant there were collaboratively-produced resource at the end
+    * I did the demotivational stories and the 'thought exercise' (challenging negative thoughts as a way of addressing impostor syndrome)
+      and on both occasions had someone decline to share with the group.
+      I prodded a bit but then let them go, which I thought was best.
```

Comment:
* Christina
   * Less "teaching" than I thought - use of exercises and so on meant that I didn't have to be "live" so much (which was great ^^)
   * Would like to incorporate more of the best practices we teach into lesson design (more + better formative assessments, mostly)
   * Not sure how to best facilitate inter-group discussion and general question asking - had at least one good inter-group discussion on etherpad chat, otherwise, seemed complicated to do over voice.  It seemed like each site was having its own discussion, which was good.